VirtoCommerce.Storefront/Domain/StaticContent/StaticContentService.cs

```diff
@@ -1,13 +1,16 @@
 using Markdig;
```

Comment:
Remove unused usings.

Automatically format documents when saving:
Install https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.PowerCommandsforVisualStudio
Go to `Tools > Options > Productivity Power Tools > PowerCommands`
Enable `Format documents on save` and `Remove and Sort Usings on save`